### DISCUSSION TITLE
Can the getUsers API search for users by ids?

### DISCUSSION THREAD

**Discussion by IRediTOTO** [logto-io/logto#6612] [Q&A] [answered]
Hi, I'm using [logto-management-api](https://bump.sh/logto/doc/logto-management-api/operation/operation-listusers ), but I need to search a list of users by ids to display on my application. Does Logto.io have any such query params? I'm looking for it and can't find it :(

**Comment by wangsijie** [comment]
You can try to use the advanced user search: https://docs.logto.io/docs/recipes/manage-users/advanced-user-search/

**Comment by wangsijie** [comment, marked as answer]
For your case, `/api/users?mode=exact&search.id=<user-id-1>&search.id=<user-id-2>`

**Reply by IRediTOTO** [reply]
:( I rollback to nextauth. Let's  me try next time...
